Update Set-DnsClientServerAddress.md (#2276)
MicrosoftDocs/windows-powershell-docs: docset/windows/dnsclient/Set-DnsClientServerAddress.md

### -ThrottleLimit
Specifies the maximum number of concurrent operations that can be established to run the cmdlet.

> If you omit this parameter or enter value of `0`, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
> The throttle limit applies only to the current cmdlet, not to the session or to the computer.

Reviewer: ```suggestion
If you omit this parameter or enter a value of `0`, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets running on the computer.
The throttle limit applies only to the current cmdlet, not to the session or the computer.
```

Reviewer 2: I would also remove the Registered mark from "Windows PowerShell".
